workers/genotype_extractor.py

```diff
@@ -21,6 +21,11 @@
 
 @worker.task(bind=True)
 def extract(self, vcf_id):
+    register_running_task(self, vcf_id)
+    return _extract(vcf_id)
```

Comment:
Can you just mock it out or something rather than creating confusing (IMO) `_` methods?